### PR TITLE
Report actual version of unsupported DWG files

### PR DESCRIPTION
**Pull request by nyalldawson**

### CODE REVIEW

File: external/libdxfrw/drw_base.h [hunk 1]
```diff
@@ -63,15 +63,24 @@ namespace DRW
...
+    AC1003,       //!< DWG Release 2.6
+    AC1004,       //!< DWG Relase 9
+    AC1006,       //!< DWG Release 10
+    AC1009,       //!< DWG Release 11/12 (LT R1/R2)
+    AC1012,       //!< DWG Release 13 (LT95)
+    AC1014,       //!< DWG Release 14/14.01 (LT97/LT98)
+    AC1015,       //!< AutoCAD 2000/200i0/2002
+    AC1018,       //!< AutoCAD 2004/2005/2006
+    AC1021,       //!< AutoCAD 2007/2008/2009
+    AC1024,       //!< AutoCAD 2010/2011/2021
```
Comment (agiudiceandrea) [inline external/libdxfrw/drw_base.h:81]:
```suggestion
    AC1015,       //!< AutoCAD 2000/2000i/2002
    AC1018,       //!< AutoCAD 2004/2005/2006
    AC1021,       //!< AutoCAD 2007/2008/2009
    AC1024,       //!< AutoCAD 2010/2011/2012
```

**Comment by nyalldawson** [comment]
Thanks @agiudiceandrea !

**Comment by lbartoletti** [comment]
Thanks @nyalldawson 

Is the fix on libdxfrw is also reported upstream [1] ?

[1] Well... where is "upstream"? https://github.com/codelibs/libdxfrw or https://github.com/LibreCAD/libdxfrw ?

**Review by alexbruy — APPROVED** [review]

**Comment by nyalldawson** [comment]
@lbartoletti I've tried to kick start discussions on a unified upstream here: https://github.com/LibreCAD/libdxfrw/issues/22

If successful I'll hunt through and submit prs from qgis' fork

**Comment by lbartoletti** [comment]
> @lbartoletti I've tried to kick start discussions on a unified upstream here: [LibreCAD/libdxfrw#22](https://github.com/LibreCAD/libdxfrw/issues/22)
> 
> If successful I'll hunt through and submit prs from qgis' fork

@nyalldawson thanks! You can rely on my help.

**Pull request merged**